libs/actions/errors.d.ts

```diff
@@ -0,0 +1,12 @@
+export function clearErrors(): {
```

Comment:
I think the idea behind the libs dir was that it would contain fully transpiled js. What's the thinking behind the change?